packages/core/src/components/date-picker/date-picker.tsx

```diff
@@ -68,7 +68,7 @@ export type DsoDatePickerKeyboardEvent = {
 }
 export type DsoDatePickerDirection = "left" | "right"
 
-const DISALLOWED_CHARACTERS = /[^0-9\.\/\-]+/g
+const DISALLOWED_CHARACTERS = /[^0-9\-]+/g
```

Comment:
Dat er nu geen `.` en `/` meer geaccepteerd worden als scheidingsteken mag wel wat expliciter in ticket worden vermeld lijkt mij.

Reply:
Ik denk niet dat het door deze change anders is geworden. De datum string wordt geparsed door parseDutchDate en daar wordt expliciet gesplit op `-`. De karakters `.` en `/` waren dus al niet toegestaan.

Het gedrag van alleen toestaan van `-0123456789` wordt nu ondersteund door een test "should only allow date characters input".